Cards/Effects/EffectBase.gd

```diff
@@ -0,0 +1,10 @@
+# Base class for card effects
+class_name EffectBase extends Node
```

Comment:
Should probably extend from Resource

Reply:
I didn't really check the difference between Node and Resources. Node is for instances ? I will check that tomorrow and change if needed

Reply:
Nodes are meant to be added to the scene tree, so they have transform data. We don't really need that for this. Also, Resources are automatically garbage collected

Reply:
Will change in next push